authors/apps/authentication/serializers.py

```diff
@@ -1,7 +1,6 @@
 from django.contrib.auth import authenticate
-
 from rest_framework import serializers
-
+import re
```

Comment:
Import this after importing the User class
```suggestion
from .models import User
import re
```